### PR TITLE
Implement BinaryOperatorExpr.Operator equal

### PR DESCRIPTION
**Pull request by Quasimewdo**
- Add BinaryExpr constructor
- Implement str() in expr.BinaryExpr
- Implement test of str() for different left
  element type
- TODO: Add query test using str()

### CODE REVIEW

File: tests/test_binaryExpr.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    assert str(b4) == "temp1.id = temp2.id"
+
+
+# TODO : Add Query Condition Test
+def test_expr_bin_equal_bool(db: gp.Database):
+    rows = [(0,), (1,), (1,), (0,)]
+    t = gp.values(rows, db=db)
+    t = t.save_as("temp1", column_names=["id"])
+    c = t["id"]
+    b5 = gp.expr.BinaryExpr("=", c, True)
```
Comment (xuebinsu) [inline tests/test_binaryExpr.py:61]: This looks quite different from what pandas provides.

File: greenplumpython/expr.py [hunk 2]
```diff
@@ -10,32 +10,67 @@ def __init__(self, as_name: Optional[str] = None) -> None:
...
 
-class BinaryExpr:
-    def __init__(self, operator: str, left: Expr, right: Expr):
-        raise NotImplementedError()
+class BinaryExpr(Expr):
+    def __init__(self, operator: str, left: Expr, right, as_name: Optional[str] = None):
+        super().__init__(as_name=as_name)
+        self.operator = operator
+        self.left = left
+        self.right = right
```
Comment (beeender) [inline greenplumpython/expr.py:45]: Those three should be private, use `_`.

**Review by beeender — APPROVED** [review]
Looks good! Except one comment

File: tests/test_column.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+def test_expr_column_str(db: gp.Database, table: gp.Table):
+    c = gp.expr.Column("id", table)
+    assert str(c) == "const_table.id"
+
+
+def test_expr_column_str_in_query(db: gp.Database, table: gp.Table):
+    c = gp.expr.Column("id", table)
+    query = "select " + str(c) + " from " + c.table.name
```
Comment (yihong0618) [inline tests/test_column.py:33]: better to use `f-string` here ?

**Review by yihong0618 — APPROVED** [review]

**Pull request merged**